pyfar/io/io.py

```diff
@@ -105,10 +105,16 @@ def convert_sofa(sofa):
 
         - :py:class:`~pyfar.classes.audio.Signal`
             A Signal object is returned is the DataType is ``'FIR'``,
-            ``'FIR-E'``, or ``'FIRE'``.
+            ``'FIR-E'``, or ``'FIRE'``. In case of ``'FIR-E'``, the time data
+            is returned with the `cshape` EMRN (samples are in the last
+            dimension) and not MRNE as in the SOFA standard (emitters are in
+            the last dimension).
```

Comment:
Mmh, I just realized that not only the position of N changes... Now I am not sure anymore

Reply:
Yes, I thought moving E to the first dimension might make most sense in python. In that case data with identical MRN-dimensions would broadcast, regardless of E. I would argue that one should know about the meaning of the dimensions if reading SOFA data. Otherwise it would be hard to understand the data.